src/services/filesync/error.ts

```diff
@@ -66,3 +66,28 @@ export class TooManySyncAttemptsError extends CLIError {
     `;
   }
 }
+
+export class DeployDisallowedError extends CLIError {
+  isBug = IsBug.NO;
```

Comment:
This has to be either `IsBug.MAYBE` or `IsBug.YES` to be sent to Sentry.

https://github.com/gadget-inc/ggt/blob/main/src/services/output/report.ts#L21-L23